pkgs/development/python-modules/pycurl/default.nix

```diff
@@ -72,6 +73,9 @@ buildPythonPackage rec {
     "test_libcurl_ssl_gnutls"
     # AssertionError: assert 'crypto' in ['curl']
     "test_ssl_in_static_libs"
+  ] ++ lib.optionals (stdenv.isDarwin || stdenv.isAarch64) [
```

Comment:
Hold on, do you mean for this to be an `&&`?

Also a random segfault is generally a bad sign.

Reply:
Good catch, that should totally be a `&&`, fixing now.

I agree it's a bad sign. I don't know how to fix it though, and at minimum this seems like a reasonable stopgap.

Reply:
Pity it's on darwin, macos seems to fight every attempt I make to debug a segfault on it.